docs/user/convert/parsing_and_validation.rst

```diff
@@ -0,0 +1,119 @@
+Parsing and Validation
+----------------------
+
+GeoMesa provides options for dealing with bad input records that can be configured on your converter. For example,
+you may want to skip over bad records for some input sources or fail entire files in other cases. The Converter
+framework provides a few options that can help control how errors in input data are handled.
+
+Validators
+~~~~~~~~~~
+
+At their core, Converters transform input streams into SimpleFeatures. Validators provide hooks to validate properties
```

Comment:
[picky style] I don't think we need to capitalize Converters unless we are talking about a Java/Scala class name, and I would spell simple features consistently (e.g. don't use both "SimpleFeatures" and "simple features").